cfn/templates/ecs-task.template.yml

```diff
@@ -0,0 +1,90 @@
+# https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-ecs-taskdefinition.html
+---
+AWSTemplateFormatVersion: 2010-09-09
+
+Description: CloudFormation template for an ECS task definition
+
+Parameters:
+  Family:
+    Type: String
+    Default: {{ family }}
+  Cpu:
+    Type: Number
+    Default: 256
+  Memory:
+    Type: Number
+    Default: 512
+  NetworkMode:
+    Type: String
+    Default: bridge
+  ExecutionRoleArn:
+    Type: String
+  # TaskRoleArn:
+  #   Type: String
+  Image:
+    Type: String
+    Default: {{ image }}
+  ContainerName:
+    Type: String
+    Default: {{ container_name }}
+Resources:
+  KlaxonScheduledTaskDefinition:
```

Comment:
I thought about this for a minute. But `rake` is just a Rails thing that executes a lot of commands. I think for it to be specific enough to help someone not super familiar with Klaxon, it'd have to be something like `KlaxonScheduledPageCheckTaskDefinition` which...is long 🤷‍♀️ but if you like that better, I can get behind it.

Reply:
Fair point. Given that the task is _technically_ just a task (the scheduled part comes from event bridge), maybe we could losed the `Scheduled` part? Otherwise I think the long name is just fine!